lib/utilities/aclUtils.js

```diff
@@ -233,4 +234,58 @@ aclUtils.sortHeaderGrants =
         return addACLParams;
     };
 
+/**
+ * convertToXml - Converts the `grantInfo` object (defined in `objectGetACL()`)
+ * to an XML DOM string
+ * @param {object} grantInfo - The `grantInfo` object defined in
+ * `objectGetACL()`
+ * @return {string} xml.join('') - The XML DOM string
+ */
+aclUtils.convertToXml = grantInfo => {
+    const { grants, ownerInfo } = grantInfo;
+    const xml = [];
+
+    xml.push('<?xml version="1.0" encoding="UTF-8" standalone="yes"?>',
+             '<AccessControlPolicy>',
+             '<Owner>',
+             `<ID>${ownerInfo.ID}</ID>`,
+             `<DisplayName>${ownerInfo.displayName}</DisplayName>`,
+             '</Owner>',
+             '<AccessControlList>'
+    );
+
+    grants.forEach(grant => {
+        xml.push('<Grant>');
+
+        // The `<Grantee>` tag has different attributes depending on whether the
+        // grant has an ID or URI
+        if (grant.ID) {
+            xml.push('<Grantee xmlns:xsi="http://www.w3.org/2001/' +
+                        'XMLSchema-instance" xsi:type="CanonicalUser">',
+                     `<ID>${grant.ID}</ID>`
+            );
+        } else if (grant.URI) {
+            xml.push('<Grantee xmlns:xsi="http://www.w3.org/2001/' +
+                        'XMLSchema-instance" xsi:type="Group">',
+                     `<URI>${escapeForXML(grant.URI)}</URI>`
+            );
+        }
+
+        if (grant.displayName) {
+            xml.push(`<DisplayName>${grant.displayName}</DisplayName>`);
```

Comment:
Good call. Vault's display names follow the regex `/^[\w+=,.@ -]+$/` and `@` should be escaped in XML.